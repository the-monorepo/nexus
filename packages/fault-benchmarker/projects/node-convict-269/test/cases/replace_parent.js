'use strict';

exports.conf = {
  top: {
    middle: {
      leaf: 'foo',
    }
  }
};
