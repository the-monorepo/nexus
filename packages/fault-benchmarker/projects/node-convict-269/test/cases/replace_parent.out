configuration param 'top.middle.leaf' missing from config, did you override its parent?
